releasenotes/notes/add-gates-to-collect-clifford-af88dd8f7a2a4bf9.yaml

```
@@ -0,0 +1,8 @@
+---
+features_transpiler:
+  - |
+    Add an argument ``matrix_based`` to the :class:`.CollectCliffords()` transpiler pass.
+    If the new parameter ``matrix_based=True``, the :class:`.CollectCliffords()` transpiler pass
+    can collect :class:`.RZGate(np.pi/2)` gates and other unitary gates that are :class:`.Clifford()`gates
+    for certain parameters.
```

```suggestion
    can collect :class:`.RZGate(np.pi/2)` gates and other unitary gates that are :class:`.Clifford()` gates
    for certain parameters.
```
Note the extra space before "gates" -- I believe this is what's responsible for CI failures :).

---

good catch!
The CI failure is not related. it's due to QPY backwards tests. should be fixed after #13202 is merged.